contracts/mocks/ERC721ReceiverMock.sol

```diff
@@ -4,8 +4,8 @@ import "../token/ERC721/ERC721Receiver.sol";
 
 
 contract ERC721ReceiverMock is ERC721Receiver {
-  bytes4 retval;
-  bool reverts;
+  bytes4 retval_;
```

Comment:
Can we make the visibility explicit here (and add a Solium rule if possible)?

Reply:
That should be a separate issue:
https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1250